grasa_event_locator/views.py

```diff
@@ -27,18 +29,126 @@ def admin(request):
 
 def changepw(request):
         return render(request, 'changePW.php')
 
+def database(request):
+        with connection.cursor() as cursor:
+                cursor.execute("delete from grasa_event_locator_category")
+                cursor.execute("ALTER TABLE grasa_event_locator_category AUTO_INCREMENT = 1")
+        table = Category(description = "Academic Support")
+        table.save()
+        table = Category(description = "Arts and Culture")
+        table.save()
+        table = Category(description = "Career or College Readiness")
+        table.save()
+        table = Category(description = "Civic Engagement")
+        table.save()
+        table = Category(description = "Community Service / Service Learning")
+        table.save()
+        table = Category(description = "Entrepreneurship / Leadership")
+        table.save()
+        table = Category(description = "Financial Literacy")
+        table.save()
+        table = Category(description = "Health & Wellness")
+        table.save()
+        table = Category(description = "Media Technology")
+        table.save()
+        table = Category(description = "Mentoring")
+        table.save()
+        table = Category(description="Nature & the Environment")
+        table.save()
+        table = Category(description="Play")
+        table.save()
+        table = Category(description="Public Speaking")
+        table.save()
+        table = Category(description="Social and Emotional Learning (SEL)")
+        table.save()
+        table = Category(description="Sports and Recreation")
+        table.save()
+        table = Category(description="STEM")
+        table.save()
+        table = Category(description="Tutoring")
+        table.save()
+        table = Category(description="Other")
+        table.save()
+        table = Category(description="Transportation Not Provided")
+        table.save()
+        table = Category(description="Transportation Provided")
+        table.save()
+        table = Category(description="K-3rd")
+        table.save()
+        table = Category(description="K-5th")
+        table.save()
+        table = Category(description="3rd-5th")
+        table.save()
+        table = Category(description="6th-8th")
+        table.save()
+        table = Category(description="9th-12th")
+        table.save()
+        table = Category(description="Not Specific")
+        table.save()
+        table = Category(description="Female")
+        table.save()
+        table = Category(description="Male")
+        table.save()
+        table = Category(description="Before School")
+        table.save()
+        table = Category(description="After School")
+        table.save()
+        table = Category(description="Evenings")
+        table.save()
+        table = Category(description="Weekends")
+        table.save()
+        table = Category(description="Summer")
+        table.save()
+        table = Category(description="Other")
+        table.save()
```

Comment:
Yikes. The repetition of `table.save()` is risky to me… but this might be something we clean up for Sprint 3.

Reply:
Oh yes, the reason for that is at present, it is nessasary to write the catagory names to the database beforehand for tags to work, this was all I could come up with. It's activated via a database button on the home page. I expect it to be replaced with the YAML file business.